README.md

```diff
@@ -5,9 +5,42 @@
 Contains the application protocol definitions for long-range devices
 connecting to [nRF Cloud](https://nrfcloud.com/). 
 
+Devices that send messages to nRFCloud can optionally have data displayed on cards. Cards will be shown only if the device shadow includes valid service ui and messages transmitted from the device use the correct protocols described in this repo.
+## General Message Schema
+
+```
+{
+   appId: AppId;
+   messageType: MessageType;
+   data?: any;
+   time?: number;
+}
+```
+
+**appId**: The application identifier assocaited with a running application on a IoT device. For example a device may be running an application sampling temperature and sending messages. Messages would include the appId TEMP.
+
+**messageType**: Below is a table of valid message types:
+
+| MessageType      | Description | deviceToCloud or cloudToDevice |
+| :-----------:      | :----------- | :-----------:                    |
+| HELLO | Tells applications that device is ready to send data	  | C2D |
```

Comment:
"HELLO", "START", "STOP", "INIT", "STATUS" and "OK" are new to me.  Do the front end or back end send these currently?  Or are they planned to be actually used in the future?

Reply:
I have no idea. This is what is documented here https://projecttools.nordicsemi.no/confluence/pages/viewpage.action?pageId=60358013.

Reply:
Those messages come from the days we thought we'd be controlling the device by sending messages (c2d) to the device. It's no longer the case anymore with ATV2. We will be using the shadow to configure the device. The messageType "DATA" is the only really important messageType, there is a backend dependency on it. I'll mark all but DATA as legacy for Atv1 just incase anyone is using it